VRDR.CLI/Program.cs

```diff
@@ -577,8 +576,12 @@ static int Main(string[] args)
 
                     if (val1.ToUpper() != val2.ToUpper() || val1.ToUpper() != val3.ToUpper() || val2.ToUpper() != val3.ToUpper())
                     {
-                        issues++;
-                        Console.WriteLine($"[***** MISMATCH *****]\t{info.Name}: {info.Contents} \t\t\"{val1}\" != \"{val2}\" != \"{val3}\"");
+                        // TEMP disable countytext feilds
+                        if (info.Name != "COUNTYTEXT_D" && info.Name != "COUNTYTEXT_I")
+                        {
+                            issues++;
+                            Console.WriteLine($"[***** MISMATCH *****]\t{info.Name}: {info.Contents} \t\t\"{val1}\" != \"{val2}\" != \"{val3}\"");
+                        }

```

Comment:
how did these changes end up in this PR? were they duplicated here or was the 351 branch merged into this one?

Reply:
The 351 branch was merged into the v1.3 branch, and then this branch was rebased on the v1.3 branch.